lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +11,11 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if type(text) != str:
+        return None
+    text = text.replace('\n', '')
```

Comment:
is this line necessary?

Reply:
I think, it is necessary, because if I delete it, then in cycle only '\' will be deleted, but we don't need extra 'n' symbols

Reply:
I think, it is necessary, because if I delete it, then in cycle only '\' will be deleted, but we don't need extra 'n' symbols

Reply:
Do you mean `'\n'.isalpha()` is going to return `True`?

Reply:
Still an issue